modules/octo-sts-secret/cmd/rotate/main.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2024 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	"cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+	"github.com/chainguard-dev/clog"
+	"github.com/kelseyhightower/envconfig"
+	"google.golang.org/api/iterator"
+
+	"github.com/chainguard-dev/terraform-infra-common/pkg/octosts"
+)
+
+type envConfig struct {
+	OctoSTSPolicy     string `envconfig:"OCTOSTS_POLICY" required:"true"`
+	GitHubOrg         string `envconfig:"GITHUB_ORG" required:"true"`
+	GitHubRepo        string `envconfig:"GITHUB_REPO" required:"true"`
+	GitHubTokenSecret string `envconfig:"GITHUB_TOKEN_SECRET" required:"true"`
+}
+
+func main() {
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		panic(fmt.Errorf("failed to parse env: %w", err))
+	}
+
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt)
```

Comment:
```suggestion
	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
```
see https://github.com/chainguard-dev/mono/pull/18076